sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2621,6 +2596,11 @@ public static CaseDataDto toDto(Case source) {
 		return target;
 	}
 
+	@Override
+	public CaseReferenceDto toRefDto(Case aCase) {
+		return convertToReferenceDto(aCase);
```

Comment:
Rename this to toReferenceDto. With this we are then finally consistent with the rest of the codebase.